pkg/components/networking/networkobservability/component.go

```diff
@@ -9,11 +9,12 @@ type Component struct {
 	*config.Component
 }
 
-var NetworkToolsComponent = Component{
+var NetworkObservabilityComponent = Component{
 	Component: &config.Component{
-		Name:                 "Networking / network-tools",
+		Name:                 "Networking / Network Observability",
```

Comment:
This OCPBUGS component doesn't even exist.  Is it something that's going to be created later? You have to wait until it exists in Jira.

Reply:
ah , okay. I actually came across "Networking / NetObs" component under OCPBUGS project and there's already package created, I'll update the Matchers for that component and remove this.